jwst/pipeline/calwebb_spec2.py

```diff
@@ -128,9 +128,9 @@ def process(self, data):
         # Setup step parameters required by the pipeline.
         self.resample_spec.save_results = self.save_results
         self.resample_spec.suffix = "s2d"
-        self.cube_build.output_type = "multi"
         self.cube_build.save_results = False
         self.cube_build.skip_dqflagging = True
+        self.cube_build.pipeline = (2,)
```

Comment:
Why is this a tuple?  I think it should just be an int.
```suggestion
        self.cube_build.pipeline = 2
```

Reply:
Not sure I think that got changed when I did not commit.  I do not remember setting it as a tuple. Let me change cause I agree